enhancing/enhancer.spec.js

```diff
@@ -1,2 +1,99 @@
+
 const enhancer = require('./enhancer.js');
-// test away!
+
+// TEST ITEM for ALL TESTS
+const elvinBow = {
+    name:'Elvin Bow',
+    durability: 90,
+    enhancement: 0,
+};
+
+describe('enhancer.js', () => {
+    //REPAIR FUNCTION
+    describe('repair(item)', () => {
+        it('should return an object with durability = 100', () => {
+            const {name,enhancement} = elvinBow;
+            expect(enhancer.repair(elvinBow)).toEqual({
+                name: name,
+                durability:100,
+                enhancement: enhancement
+            });
+        });
+    });
+
+    //SUCCEED FUNCTION
+    describe('succeed(item)', () => {
+        const {name,durability,enhancement} = elvinBow;
+        if (enhancement < 20) {
+            it('should return an object with enhancement+1', () => {
+                expect(enhancer.succeed(elvinBow)).toEqual({
```

Comment:
not necessary, but don't forget you could destructure all the enhancer functions at the top of the file so you don't have to keep repeated `enhancer.` for each test